hazelcast/src/main/java/com/hazelcast/client/config/ClientReliableTopicConfig.java

```diff
@@ -84,8 +85,9 @@ public String getName() {
      * Sets the name or name pattern for this config. Must not be modified after this
      * instance is added to {@link ClientConfig}.
      */
-    public void setName(String name) {
+    public NamedConfig setName(String name) {
```

Comment:
looks like this could be `ClientReliableTopicConfig` (similar to the way done below for `FlakeIdGeneratorConfig`)

Reply:
Made by mistake. I am fixing it.